package.json

```diff
@@ -4,21 +4,24 @@
   "description": "Project to go over fundamentals",
   "main": "lib/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
     "dev": "nodemon --exec babel-node lib/index.js",
     "build": "babel lib --out-dir build",
     "start": "node build/index.js"
   },
   "author": "Jordan Kelly",
   "license": "ISC",
   "dependencies": {
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "path": "^0.12.7"
```

Comment:
`path` is a standard node library. We should be able to include it, without it being in our deps.